packages/components/pages/f-contact-preferences/package.json

```diff
@@ -56,15 +56,20 @@
   },
   "peerDependencies": {
     "@justeat/browserslist-config-fozzie": ">=1.2.0",
+    "@justeat/f-alert": "6.x",
+    "@justeat/f-button": "4.x",
+    "@justeat/f-card": "4.x",
+    "@justeat/f-card-with-content": "3.x",
+    "@justeat/f-form-field": "6.x",
     "vuex": ">=3.5.1"
   },
   "devDependencies": {
+    "@justeat/f-alert": "6.x",
+    "@justeat/f-button": "4.x",
+    "@justeat/f-card": "4.x",
+    "@justeat/f-card-with-content": "3.x",
+    "@justeat/f-form-field": "6.x",
     "@justeat/f-vue-icons": "3.3.0",
```

Comment:
Any reason for leaving this?

Reply:
My thinking was that I'd try and keep the complexity down to start with by sticking only to components for now